bitriseio/finish.go

```diff
@@ -0,0 +1,45 @@
+package bitriseio
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// RegisterFinishParams ...
+type RegisterFinishParams struct {
+	Config           string            `json:"config,omitempty"` // in case of local run we do not have scan result, so our only option is Mode: manual + Config: any default config id
+	Envs             map[string]string `json:"envs,omitempty"`
+	Mode             string            `json:"mode,omitempty"`
+	OrganizationSlug string            `json:"organization_slug,omitempty"` // leave it empty if the user will be the owne
+	ProjectType      string            `json:"project_type,omitempty"`
+	StackID          string            `json:"stack_id,omitempty"`
+}
+
+// RegisterFinishResponse ...
+type RegisterFinishResponse struct {
+	Status                    string `json:"status,omitempty"`
+	BuildTriggerToken         string `json:"build_trigger_token,omitempty"`
+	BranchName                string `json:"branch_name,omitempty"`
+	IsWebhookAutoRegSupported bool   `json:"is_webhook_auto_reg_supported,omitempty"`
+}
+
+// RegisterFinishURL ...
+func RegisterFinishURL(appSlug string) string {
+	return fmt.Sprintf(AppsServiceURL+"%s/finish", appSlug)
+}
+
+// RegisterFinish ...
+func (s *AppsService) RegisterFinish(appSlug string, params RegisterFinishParams) (*RegisterFinishResponse, error) {
+	params.Mode = "manual"
+	params.Config = "default-ios-config"
```

Comment:
is it intended?

Reply:
There's a comment in the code: `in case of local run we do not have scan result, so our only option is Mode: manual + Config: any default config id` (https://github.com/bitrise-io/bitrise-add-new-project/pull/15/files#diff-2e0865cbdec6dba0af6579f53dbf8be4R10)

However, I also stumbled here, so this might signal, that the comment at the type definition is too far away! Maybe it could be put here instead?